source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -0,0 +1,158 @@
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Section Title in Title Case 🐹](#section-url)
+
+<change section title emoji>
+<consider adding some bold to your paragraph>
+
+<add your name to author list, top and bottom>
+<add blurb and emoji to "SOME-INTRO-HERE">
+
+---
+
+## [Where Can I Learn More? 🤓](https://guides.emberjs.com/release/)
+
+If you want to try out Ember Octane, you might want to learn more about the new APIs and best practices for
+building modern Ember apps. In this regard we highly recommend the following learning resources to you.
+
+### Getting Started
+
+- [The official documentation - Ember.js Guides](https://guides.emberjs.com/release/)
+- [The official beginner's tutorial - Build the Super Rentals app](https://guides.emberjs.com/release/tutorial/part-1/)
+- [The official API reference - Ember.js API Docs](https://api.emberjs.com/ember/release)
+
+### Octane Core Concepts
+
+- Octane Core Concepts Blog Series by [Chris Garrett (@pzuraq)](https://github.com/pzuraq)
+  - [Part 1: Native Classes](https://blog.emberjs.com/2019/02/11/coming-soon-in-ember-octane-part-1.html)
+  - [Part 2: Angle Brackets & Named Arguments](https://blog.emberjs.com/2019/02/19/coming-soon-in-ember-octane-part-2.html)
+  - [Part 3: Tracked Properties](https://blog.emberjs.com/2019/02/26/coming-soon-in-ember-octane-part-3.html)
+  - [Part 4: Modifiers](https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html)
+  - [Part 5: Glimmer Components](https://blog.emberjs.com/2019/03/14/coming-soon-in-ember-octane-part-5.html)
+- [Most Common Mistakes Using Octane and How to Avoid Them](https://medium.com/ember-ish/the-most-common-ember-js-octane-mistakes-and-how-to-avoid-them-c6420e1b0423) by [Jen Weber (@jenweber)](https://github.com/jenweber)
+- [Bringing Clarity to Templates through Ember Octane](https://simplabs.com/blog/2019/12/20/clarity-in-templates/) by [Ricardo Mendes (@locks)](https://github.com/locks)
+- [Ember Octane Fundamentals Course on Frontend Masters (paid subscription)](https://frontendmasters.com/courses/ember-octane/)
+
+### Migrating Ember Apps
+
+- [Rewriting Apps in Ember Octane](https://crunchingnumbers.live/2019/12/23/rewriting-apps-in-ember-octane/) by [Isaac Lee (@ijlee2)](https://github.com/ijlee2)
+- [The Ember Octane vs. Classic Ember Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)
```

Comment:
it would be great to point out all the available codemods in place as well as the recommended order in which folks should start their migrations.
- https://github.com/ember-codemods
- The Atlas - https://www.notion.so/Ember-Octane-b8ad64b7adf74218942fc69ead7b6892 which contains the recommended migration order too (https://www.notion.so/Atlas-Recommended-Migration-Order-a22f948a7cce4e01896d674f727bee74)

Reply:
Thanks @suchitadoshi1987 will defs add these in - we're still fleshing out this section :)

Reply:
@suchitadoshi1987 thanks for the feedback, this is included! I'm looking over the Ember Atlas rn to flesh out some more on a separate blurb there as well. Definitely keep the comments coming or join us in #support-ember-times on Discord anytime!